features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1058,7 +1058,7 @@
   "console.ui.administrator_role_display_name": "Administrator",
   "console.ui.legacy_mode.apiResources": true,
   "console.ui.legacy_mode.applicationListSystemApps": false,
-  "console.ui.legacy_mode.applicationOIDCSubjectIdentifier": false,
+  "console.ui.legacy_mode.applicationOIDCSubjectIdentifier": true,
```

Comment:
Can this be considered legacy mode now?
We can remove this config right?

Reply:
Yes.. let's remove the config in a follow-up PR